content/articles/understanding-shared-memory-programming-with-pthreads-and-openmp/index.md

```diff
@@ -0,0 +1,308 @@
+### Introduction
+Shared memory is used to speed up data flow between programs. Depending on the circumstance, programs may operate on one or more processors. As a result, a process may have several threads. Threads are referred to as `lightweight processes.` They are referred to as shared lightweight processes since they share the same address space and other resources, unlike processes that do share addresses.
+
+Parallelism is achieved by threading a process. Parallelism is a state where many processes are carried out simultaneously. Browser tabs, for example, are threads. Likewise, MS Word uses threads to format text, handle input, and many other things. 

```

Comment:
A reader maybe confused here, in one instance you talk of threads, in another, you talk of processes being threaded.